rclcpp/include/rclcpp/node_interfaces/node_base.hpp

```diff
@@ -43,7 +43,7 @@ class NodeBase : public NodeBaseInterface
     rclcpp::Context::SharedPtr context,
     const rcl_node_options_t & rcl_node_options,
     bool use_intra_process_default,
-    bool enable_topic_statistics_default);
+    bool enable_topic_statistics_default = false);
```

Comment:
Why make this optional again?

Reply:
Think this is a rebase (merge conflicts) mistake. Will revert.